theseus/src/state/mod.rs

```diff
@@ -345,27 +345,25 @@ pub async fn init_watcher() -> crate::Result<Debouncer<RecommendedWatcher>> {
     let (mut tx, mut rx) = channel(1);
 
     let file_watcher = new_debouncer(
-        Duration::from_secs_f32(2.0),
+        Duration::from_secs_f32(10.0),
```

Comment:
10 seconds is very long for the profile syning, mods will take forever to add. what was the reason for this change?

Reply:
was trying to reduce cpu load on installs, (when many packs are imported one after another). Reverted as I don't think this is the biggest impact change I made